openshift/cronJob-template.yaml

```diff
@@ -69,7 +69,6 @@ objects:
                 app: thoth
                 component: graph-backup
             spec:
-              serviceAccountName: "graph-backup-job"
```

Comment:
In the stage environment are we going to deploy this in `thoth-infra-stage` ? 
or else it would need some sa to pull image from infra, i think.

Reply:
I've deployed the cronjob in thoth-frontend-stage, it is using `docker-registry.default.svc:5000/thoth-infra-stage/graph-backup-job:latest`, seems like its operational and can pull the image.

Reply:
I had already created sa in thoth-frontend-stage for the graph-backup-job.
https://github.com/thoth-station/thoth-ops/blob/7c677381a335cd91519e2c8b437369ccf0cc281e/ansible/provision_environment.yaml#L188
If we remove the sa from buildconfig, then it would start using the default sa, we can setup policies for that if we don't want to explicitly create an sa for the module.
WDYT @fridex ?

Reply:
Originally I removed SA as the backup job was failing due to missing SA:

```
$ oc get sa -n thoth-frontend-stage graph-backup-job                            
No resources found.
Error from server (NotFound): serviceaccounts "graph-backup-job" not found
```

I would say let's keep the default one if it is sufficient for the job to run.

Reply:
Alright! I will merge the pr 😁